reader/utils/getArticleMetadata.js

```diff
@@ -0,0 +1,74 @@
+import axios from 'axios'
+
+export const CORE_API = 'https://core.ac.uk:443/api-v2/articles/get'
+
+const apiRequest = async (url, params = {}, method = 'GET') => {
+  try {
+    const response = await axios({
+      url,
+      method,
+      params,
+      headers: {
+        'Content-Type': 'application/json',
+        apiKey: CORE_API_KEY,
+      },
+    })
+
+    let { data } = response.data
+    // TODO: These codes should return API
+    // status (string) = ['OK' or 'Not found' or 'Too many queries' or
+    // 'Missing parameter' or 'Invalid parameter' or 'Parameter out of bounds']
+    switch (response.data.status) {
```

Comment:
Honestly, our API is * (not good). We definitely have to fix this in next version.